package.json

```diff
@@ -14,6 +14,7 @@
     "@emotion/server": "latest",
     "@emotion/styled": "latest",
     "@mui/icons-material": "latest",
+    "@mui/lab": "^5.0.0-alpha.79",
```

Comment:
Didn't realize this was part of the "lab". I don't think it matters, but just weird that their docs don't mention it on the Button page.